exercises/J-array-get-set/exercises2.js

```diff
@@ -6,8 +6,9 @@
   - change the first value in the array to the number 1
 */
 
-let numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+let numbers = [5, 2, 3]; // Don't change this array literal declaration
+numbers[3] = 4;
```

Comment:
Nice, this works. What [array method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#static_methods) can we use to add things to the end of an array?

Reply:
.push method

Reply:
Perfect